doc/build-unix.md

```diff
@@ -166,6 +156,12 @@ libqrencode (optional) can be installed with:
 Once these are installed, they will be found by configure and a bitcoin-qt executable will be
 built by default.
 
+## Dependencies
+
+See [dependencies.md](dependencies.md) for a complete overview, and
+[depends](/depends/README.md) on how to compile them yourself, if you wish to
+not use the packages of your Linux distribution.
+
 Notes
```

Comment:
While here, I think you could also remove the `Notes`, as well as the `miniupnpc` and `libnatpmp` sections below. I don't think `Notes` really adds any value, and the  `miniupnpc` and `libnatpmp` seem irrelevant here (we also don't have a similar section for all other dependencies (nor do I think we should add one), and listing configure defaults in markdown is mostly pointless, and just duplicates the (always up to date) output of `./configure --help`).

We could also remove the `Security` section further down, I don't think this document is the right place for generic, binary hardening information.

Reply:
Didn't drop `Security` ?

Reply:
Ok. Done in #27688.